### PR TITLE
[stable/kibana] Support custom labels on Service

### PR DESCRIPTION
**Pull request by NicolasT**
In line with various other charts, allow to set custom labels on the rendered `Service` object by setting values.

### CODE REVIEW

**Comment by NicolasT** [comment]
CC @compleatang

**Comment by NicolasT** [comment]
CLA has been signed, I believe.

**Comment by NicolasT** [comment]
CC @JessicaGreben 
/assign @viglesiasce

File: stable/kibana/templates/service.yaml [hunk 1]
```diff
@@ -6,6 +6,9 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- range $key, $value := .Values.service.labels }}
```
Comment (compleatang) [inline stable/kibana/templates/service.yaml]: in keeping with helm defaults, can you outdent this line and the `{{- end }} line two spaces please? in general this makes the chart more readable.

**Review by compleatang — CHANGES_REQUESTED** [review]

**Comment by compleatang** [comment]
@NicolasT I added one minor change request. Also, can you please add a line in the README so that folks understand the functionality exists without having to parse the values.yaml file? 

Thanks!

**Comment by compleatang** [comment]
LGTM

**Comment by NicolasT** [comment]
Anything I can do to make this move forward?

**Comment by unguiculus** [comment]
Your branch is outdated. You need to merge in the current master so CI works.

**Pull request merged**